exporter/prometheusremotewriteexporter/exporter.go

```diff
@@ -30,6 +30,7 @@ import (
 	"github.com/gogo/protobuf/proto"
 	"github.com/golang/snappy"
 	"github.com/prometheus/prometheus/prompb"
+	"github.com/tidwall/wal"
```

Comment:
Should we be using github.com/prometheus/prometheus/tsdb/wal?

Reply:
I believe we talked about this in meetings a few times, the Prometheus WAL needs a Go struct-based prometheus.Registerer which would involve converting from OTLP->Prometheus Go then from Prometheus Go->PrometheusProto which is unnecessary roundtripping.

Reply:
github.com/prometheus/prometheus/tsdb/wal is actually a generic WAL implementation which takes byte slices (similar to tidwall/wal but without the index). It would be interesting (and looking forward!) to see performance of `tidwall/wal` with remote write :)